app/views/planning_applications/review/reports/show.html.erb

```diff
@@ -25,30 +25,106 @@
   </div>
 </div>
 
+<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
 <%= render BopsCore::TableOfContentsComponent.new(items: [
-      ## a section with no heading
-      # {links: [
-      #  ["foo", "#"]  # links are arrays containing two items, text and href
-      # ]},
-      ## sections with headings
-      # {title: "first", links: [
-      #  ["foo", "#"]
-      # ]},
-      # {title: "second", links: [
-      #  ["bar", "#"]
-      # ]}
+      {links: [["Pre-application outcome", "#pre-application-outcome"]]},
+      {links: [["Your pre-application details", "#pre-application-details"]]}
```

Comment:
This will create two lists with one item each which might be weird for screenreaders. It might be better to combine them into a single list:

```suggestion
      {links: [
        ["Pre-application outcome", "#pre-application-outcome"],
        ["Your pre-application details", "#pre-application-details"]
      ]}
```